fmriprep/workflows/bold/base.py

```diff
@@ -453,6 +445,30 @@ def init_func_preproc_wf(bold_file, ignore, freesurfer,
         (summary, func_reports_wf, [('out_report', 'inputnode.summary_report')]),
     ])
 
+    # if multiecho data, select middle echo for hmc correction
```

Comment:
This is something I have overlooked so far. Talking today with Chris I learned that the order of echoes (`echo-[1-3]`) may not align with the acquisition order. Therefore, selecting the first echo does not ensure that you are using a particular echo time.

Additionally, the comment says middle echo but the interface selects the first. Am I correct?

(I think this is the very last comment from me)